includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1060,6 +1064,23 @@ protected function manage_customer_details_for_order( $order ) {
 		return [ $user, $customer_id ];
 	}
 
+	/**
+	 * Update the saved payment method information with checkout values.
+	 *
+	 * @param string $payment_method The payment method to update.
+	 * @param string $order_id       WC order id.
+	 */
+	public function update_saved_payment_method( $payment_method, $order_id ) {
```

Comment:
May prefix `async_` for this function too.

Reply:
The function is just updating the method, it is being invoked from an async route but I don't think that should be reflected in the function name itself.